tests/databases/constructed_molecule/fixtures/constructed_molecule_mongo_db.py

```diff
@@ -98,7 +98,7 @@
                         ),
                     ),
                 ),
-                lru_cache_size=128,
+                lru_cache_size=0,
```

Comment:
why? also - even if I did agree to this, it would have to be a separate PR. I'm guessing this was added here by mistake?

Reply:
I do not even know what that does and did not change it. mustve been a copy and paste error...